dev/bots/README.md

```diff
@@ -87,3 +87,46 @@ Support for [cross-repository
 recipes](https://github.com/luci/recipes-py/blob/master/doc/cross_repo.md) is
 in-progress.  If you view the git log of this directory, you'll see we initially
 tried, but it's not quite ready.
+
+# Android Tools
+
+Instructions to update the Andoird Tools version that the bots download by executing `download_andoird_tools.py`.
+
+## How to update Android SDK on GCS
```

Comment:
Oh Google Cloud Storage. We should expand the acronym here and in the heading below.